### ISSUE TITLE
Make instances of chezmoi templates comments

### ISSUE THREAD

**Issue by tapayne88** [tapayne88/dotfiles#67]
Make chezmoi template files happier (LSP / formatting / syntax) by putting instances of chezmoi template literals `{{ .foo }}` in a comment string.

Instances like this

https://github.com/tapayne88/dotfiles/blob/4e7042d12d9783217c2d575693867f5391a9dc92/dot_config/nixpkgs/modules/neovim.nix.tmpl#L16-L20

**Comment by tapayne88** [comment]
Did a quick search for possible instances
```console
❯ fd -g '*.tmpl'                                                                                                                                                                                                                         
dot_config/alacritty/alacritty.yml.tmpl
dot_config/create_private_hub.tmpl
dot_config/kitty/kitty.conf.tmpl
dot_config/nixpkgs/modules/neovim.nix.tmpl
dot_config/nvim/lua/tap/plugins/fugitive.lua.tmpl
dot_config/nvim/lua/tap/settings.lua.tmpl
dot_config/zsh/functions/wsl.zsh.tmpl
dot_config/zsh/lib/aliases.zsh.tmpl
dot_config/zsh/patches/omz.zsh.tmpl
dot_gitconfig-work.tmpl
dot_gitconfig.tmpl
dot_local/bin/executable_nix-update.tmpl
dot_local/share/applications/alacritty.desktop.tmpl
dot_local/share/applications/kitty.desktop.tmpl
dot_tmux.conf.tmpl
dot_zshrc.tmpl
misc/windows-terminal-settings.json.tmpl
private_dot_ssh/create_private_config.tmpl
```

**Comment by tapayne88** [comment]
Fixed with #69
